### PR TITLE
Fix duplicates

### PR DESCRIPTION
**Pull request by govekk**
New importer code:

1. Reduces building duplicates by making siemens buildings same as lucid
2. Improves turbine naming by cutting off only last "(" and running strip
3. Adds descriptions to point types
4. Fixes error where points weren't getting equipment ids

### CODE REVIEW

File: src/datareaders/lucid/lucid_reader.py [hunk 1]
```diff
@@ -38,11 +38,12 @@ def add_points(self):
...
             try:
                 name, building_name, description = point_names[i].split(" - ")
+                name = name + " - " + "(".join(description.split("(")[:-1])
             except ValueError:
-                name = building_name + " - " + description.split("(")[0]
-
-            if "Old Meter" in description:  # We need to differentiate between old meters and new ones.
-                name = name + "(Old Meter)"
+                building_name, description = point_names[i].split(" - ")
+                name = building_name + " - " + "(".join(description.split("(")[:-1]) # remove units information
```
Comment (carolynryan20) [inline src/datareaders/lucid/lucid_reader.py]: How is this removing units information? What does description look like before this line?

Comment (carolynryan20) [reply src/datareaders/lucid/lucid_reader.py]: Is it just separating units and then we ignore parentheses parsing for units?

Comment (govekk) [reply src/datareaders/lucid/lucid_reader.py]: Description might look like "Steam (kBTU)" or "Steam (Old Meter) (kBTU)"

**Review by carolynryan20 — APPROVED** [review]
👍

File: src/datareaders/lucid/lucid_reader.py (hunk 1 above)
Comment (carolynryan20) [reply src/datareaders/lucid/lucid_reader.py]: I see! Thanks

**Pull request merged**